topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,414 @@
+            
+requirements:
+
+   -
+   
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+           - metaquantome-data-creation
+            
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+metaQuantome software suite {% cite Easterly2019 %} was developed by the {% cite Galaxy-P %} for quantitative and statistical analysis of metaproteomics data. For taxonomic and functional expression analysis within the microbial community, metaQuantome leverages peptide-level quantitative information to generate visual outputs for data interpretation. It also generates outputs that help in understanding the taxonomic contribution to a selected function as well as functions expressed by selected taxonomic group. 
+
+In this tutorial, we will learn specifically about the metaQuantome Function workflow. In particular, we will learn about how peptide-level quantitation and associated functional information can be used to generate bar plots (for functional information such as molecular function, cellular components and biological processes), volcano plots (to detect differentially expressed function) and heatmap cluster analysis. To demonstrate the use of this workflow, we have used a thermophilic biogas reactor dataset wherein municipal food waste and manure is digested to generate methane gas ({% cite Delogu2020 %}). After one round in the reactor, the microbial community was simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen three time points (8 hours, 18 hours and 33 hours) from this dataset.
+
+![MetaQuantome-function-workflow](../../images/mQ-function-workflow.jpg "MetaQuantome workflow showing steps to generate visualisations and analysis results from input data."){: width="100%"}
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# **Pretreatments**
+
+The first step in this tutorial is to get the data from the Zenodo link provided and make sure that it is in the correct format.
+
+
+## *Get data*
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial and give it a meaningful name
+>
+>    {% include snippets/create_new_history.md %}
+>    {% include snippets/rename_history.md %}
+>
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}): a Functional File and an Intensity file.
+>  
+>
+>    ```
+>    https://zenodo.org/record/4110725/files/Function-File.tabular
+>    https://zenodo.org/record/4110725/files/Intensity-File.tabular
+>
+>    ```
+>    {% include snippets/import_via_link.md %}
+>    Alternatively, import the files from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`)
+>
+>
```

Comment:
```suggestion
>    {% include snippets/import_via_link.md %}
>
>    Alternatively, import the files from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`)
>
>    {% include snippets/import_from_data_library.md %}
>
>
```

Reply:
You can use the `{% include snippets/import_from_data_library.md %}` snippet here with instructions for loading files from a shared data libray.